### ISSUE TITLE
Google drive root not deleting files as there were IO errors

### ISSUE THREAD

**Issue by RaXorX** [kapitainsky/RcloneBrowser#124]
Having this weird issue with my tasks. My tasks used to run fine when I created them, and I even used them before multiple times for days. I haven't touched those tasks since I wasn't working on those things. 

Anyways, I have this weird issue now that through the task, rclone isn't detecting the source directory and then also gives that weird issue above. 

`2020/06/05 20:41:28 ERROR : : error reading source directory: directory not found
2020/06/05 20:41:28 ERROR : Google drive root 'Sample': not deleting files as there were IO errors
2020/06/05 20:41:28 ERROR : Google drive root 'Sample': not deleting directories as there were IO errors
2020/06/05 20:41:28 INFO  : There was nothing to transfer
2020/06/05 20:41:28 ERROR : Attempt 1/5 failed with 1 errors and: directory not found
2020/06/05 20:41:28 INFO  :
Transferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Elapsed time:         0.0s

2020/06/05 20:41:28 ERROR : : error reading source directory: directory not found
2020/06/05 20:41:28 ERROR : Google drive root 'Sample': not deleting files as there were IO errors
2020/06/05 20:41:28 ERROR : Google drive root 'Sample': not deleting directories as there were IO errors
2020/06/05 20:41:28 INFO  : There was nothing to transfer
2020/06/05 20:41:28 ERROR : Attempt 2/5 failed with 1 errors and: directory not found
2020/06/05 20:41:29 ERROR : : error reading source directory: directory not found
2020/06/05 20:41:29 ERROR : Google drive root 'Sample': not deleting files as there were IO errors
2020/06/05 20:41:29 ERROR : Google drive root 'Sample': not deleting directories as there were IO errors
2020/06/05 20:41:29 INFO  : There was nothing to transfer
2020/06/05 20:41:29 ERROR : Attempt 3/5 failed with 1 errors and: directory not found
2020/06/05 20:41:29 INFO  :
Transferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Elapsed time:         0.0s

2020/06/05 20:41:30 ERROR : : error reading source directory: directory not found
2020/06/05 20:41:30 ERROR : Google drive root 'Sample': not deleting files as there were IO errors
2020/06/05 20:41:30 ERROR : Google drive root 'Sample': not deleting directories as there were IO errors
2020/06/05 20:41:30 INFO  : There was nothing to transfer
2020/06/05 20:41:30 ERROR : Attempt 4/5 failed with 1 errors and: directory not found
2020/06/05 20:41:30 INFO  :
Transferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Elapsed time:         0.0s

2020/06/05 20:41:31 ERROR : : error reading source directory: directory not found
2020/06/05 20:41:31 ERROR : Google drive root 'Sample': not deleting files as there were IO errors
2020/06/05 20:41:31 ERROR : Google drive root 'Sample': not deleting directories as there were IO errors
2020/06/05 20:41:31 INFO  : There was nothing to transfer
2020/06/05 20:41:31 ERROR : Attempt 5/5 failed with 1 errors and: directory not found
2020/06/05 20:41:31 INFO  :
Transferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                 1 (retrying may help)
Elapsed time:         0.0s

2020/06/05 20:41:31 Failed to sync: directory not found`

This is the output. Whereas if I simply run the upload function, there's no error at all. 

`2020/06/05 20:48:50 NOTICE: Test1.txt: Not copying as --dry-run
2020/06/05 20:48:50 NOTICE: Test2.txt: Not copying as --dry-run
2020/06/05 20:48:50 NOTICE: Test.txt: Not copying as --dry-run
2020/06/05 20:48:50 INFO  :
Transferred:   	         0 / 0 Bytes, -, 0 Bytes/s, ETA -
Transferred:            3 / 3, 100%
Elapsed time:         0.0s
`

My upload task used to work fine before. And manual upload to the same Sample directory runs fine. I'll try and create new tasks and see if the issue is still there or what.

**Comment by kapitainsky** [comment]
Thanks for reporting. I will investigate asap.

**Comment by RaXorX** [comment]
Huh, just created a new task with the same settings and it's working fine. Weird. I came from 1.8.0 to the current beta. And gosh, that was a super quick reply kapitainsky 👌

**Comment by kapitainsky** [comment]
have you upgraded to the latest 1.52 rclone by any chance?

**Comment by RaXorX** [comment]
Yes I have. And sorry, false alarm. My old tasks actually had a different path due to user account change. I must have created it on another installation and then changed/formatted and created a new name. Although, once I fixed the file path, I had to quit rclone browser and restart it up. It worked fine afterwards
